env.d.ts

```diff
@@ -47,3 +47,22 @@ declare type ProcessEnvCustomKeys =
   | 'BREVO_API_KEY'
   | 'MATOMO_URL'
   | 'MATOMO_TOKEN'
+
+type UnionToIntersection<U> = (U extends any ? (k: U) => void : never) extends (
```

Comment:
🤯 ça c'est du typage costaud ; tu ferais pas l'advent of code toi à tout hasard ?

Reply:
Eheh. J'ai mal au crane mais je cherche encore un moyen plus aisé de réduire un union type...